projects/portal/src/app/pipes/unit-conversion.pipe.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { UnitConversionPipe } from './unit-conversion.pipe';
+
+describe('UnitConversionPipe', () => {
+  const pipe = new UnitConversionPipe();
+
+  it('transform string number to K units', () => {
+    expect(pipe.transform('1234')).toBe('1.23K');
+  });
+
+  it('transform number to M units', () => {
+    expect(pipe.transform(1234567)).toBe('1.23M');
+  });
+
+  it('transform string number to G units', () => {
```

Comment:
Use upper and lower boundary values.

Reply:
Arranged by unit and number of digits. 
Then added the case of the lower boundary.